pkg/cli/images.go

```diff
@@ -103,26 +138,28 @@ func printContainerImages(images []apiv1.Image, ctx context.Context, c client.Cl
 	}
 
 	for _, image := range images {
-		if image.Tag == "" && image.Repository == "" {
-			if !a.All {
-				continue
-			}
-			if !a.Quiet {
-				image.Tag = "<none>"
-				image.Repository = "<none>"
-			} else {
-				image.Tag = image.Name
-				image.Repository = "<none>"
-			}
-		}
-
 		containerImages, err := getImageContainers(c, ctx, image)
 		if err != nil {
 			return err
 		}
 
 		for _, imgContainer := range containerImages {
-			out.Write(imgContainer)
+			imageContainerPrint := imageContainerPrint{
+				Container: imgContainer.Container,
+				Repo:      imgContainer.Repo,
+				Digest:    imgContainer.Digest,
+				ImageID:   imgContainer.ImageID,
+			}
+			for _, tag := range imgContainer.Tags {
+				imageContainerPrint.Tag = tag
+				out.Write(imageContainerPrint)
+			}
+			if len(imgContainer.Tags) == 0 && a.All {
+				imageContainerPrint.Tag = "<none>"
+				imageContainerPrint.Repo = "<none>"
+				out.Write(imageContainerPrint)
+			}
+
```

Comment:
I wanted to put an emphasis on this as this seems like a regression in behavior. Before we wouldn't print out `<none>` if `--quiet` is enabled, even if `--all` is set. With this change we will always print out `<none>` regardless of if `--quiet` is set unless I'm missing something?

Reply:
I think it is working as expected here is the output if I have one image tagged and one untagged 

```
jsilverio@joshuas-mbp acorn % ./bin/acorn image   
REPOSITORY   TAG       IMAGE-ID
test/test    v0.0.1    6f0ad805c830
jsilverio@joshuas-mbp acorn % ./bin/acorn image -a
REPOSITORY   TAG       IMAGE-ID
<none>       <none>    f84d1983a6e0
test/test    v0.0.1    6f0ad805c830
jsilverio@joshuas-mbp acorn % ./bin/acorn image -q
6f0ad805c8300ec01e270bc2fe73962a3894b080e803de799952575133e578a9
jsilverio@joshuas-mbp acorn % ./bin/acorn image -a -q 
f84d1983a6e0a1af73a588a1468feefba9656d8087dd8d664c72e784c7206ac0
6f0ad805c8300ec01e270bc2fe73962a3894b080e803de799952575133e578a9
jsilverio@joshuas-mbp acorn % 
```

Reply:
Hm, interesting. I'll mark this as resolved so don't worry about replying soon, but any idea what's causing that to work as expected?